cypress/integration/utils/image-segmentation-editor-test-part/RightSideBar.js

```diff
@@ -0,0 +1,111 @@
+import goToEditor from "../go-to-editor"
+import clickLeftSideBar from "../click-on-left-sidebar"
+const tests = () => {
+  it("See the class for the annotation", () => {
+    goToEditor("Image Segmentation", 0)
+    cy.get(
+      "button[class='MuiButtonBase-root WithStyles(ForwardRef(ButtonBase))-root-157']"
+    ).click()
+    cy.get(
+      "button[class='MuiButtonBase-root MuiIconButton-root makeStyles-expandButton-144']"
```

Comment:
again, too tightly coupled, we don't want a developer trying to contribute to have to figure these class names if they make a style change.

Reply:
Check above